Changes

```diff
@@ -1,6 +1,23 @@
 Release history for Zonemaster component Zonemaster-Backend
 
 
+v11.0.0 2023-06-21 (public release version)
+
+ [Breaking changes]
+ - Removes deprecated features related to locale (#1097)
+
+ [Features]
+ - Adds new experimental API methods and method names
+   (#1111, #1096, #1083)
```

Comment:
#1054 is missing (not all the commits from this have been reverted)

Reply:
Fixed.